pages/code/[problemId].tsx

```diff
@@ -101,6 +101,10 @@ export default function CodeEditor() {
           <ProblemView
             problemTitle={currentProblem?.title}
             problemStatement={currentProblem?.statement}
+            problemAuthor={currentProblem?.author}
```

Comment:
This doesn't have to do with this PR specifically but since all of these props come from currentProblem, we should switch to just passing down currentProblem, not each attribute individually